modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -322,6 +349,9 @@ protected void containerIsStarted(InspectContainerResponse containerInfo) {
                 createBucket(bucket.getBucketSettings(), bucket.getUserSettings(), primaryIndex);
             }
         }
+        if (fts && ftsName != null && !ftsName.isEmpty()) {
```

Comment:
I don't think the guard is needed like this. IMO checking `ftsName != null` is basically fine here.
If you want to validate correct use of the setter, probably checking for empty String would make sense in the setter, although I'm not sure how we handle in other containers.